### PR TITLE
docs: Added dark theme to documentation

### PR DESCRIPTION
**Pull request by Brettm12345**
I added a dark theme to the starship documentation. Currently it only supports the `prefers-colorscheme` media query with no manual toggle

### CODE REVIEW

**Comment by matchai** [comment]
Woah, great idea!
I'm amazed how easy it is to add dark theme to a vuepress site!

The only issue I see is the accent color for the dark theme being too bright, making the call-to-action hard to read with too low a contrast-ratio:

image
image

Unfortunately the default color is too dark on black:

image
image

If we can force the CTA button to keep the default accent color, it looks like this should be good to merge. 😊

**Comment by Brettm12345** [comment]
I brought it up to 4.57

**Comment by Brettm12345** [comment]
Also here is how I configured Starship for nixos
https://github.com/Brettm12345/nixos-config/blob/master/modules/applications/prompt.nix

File: docs/.vuepress/styles/palette.styl [hunk 1]
```diff
@@ -1,4 +1,5 @@
 $accentColor = #DD0B78
+$accentDarkColor = ##e50a6c
```
Comment (matchai) [inline docs/.vuepress/styles/palette.styl]: It looks like the site isn't building due to the extra hash here:
```suggestion
$accentDarkColor = #e50a6c
```

**Comment by Brettm12345** [comment]
Fixed. My bad

**Pull request merged**